src/content/docs/tutorial-optimize-telemetry/detect-anomalies.mdx

```diff
@@ -0,0 +1,102 @@
+---
+title: "Detect ingest anomalies"
+metaDescription: Use your baseline report to detect ingest anomalies.
+redirects:
+---
+
+import omaoedgIngestTargetLine from 'images/oma-oe-dg_screenshot-full_ingest-target-line.webp'
+
+import omaoedgLookoutViewConsumingAccount from 'images/oma-oe-dg_screenshot-crop_lookout-view-consuming-account.webp'
+
+Once you've installed the baseline dashboard `link here`, you should keep an eye on it to detect any ingest anomalies as often as possible. While you have to wait until you have enough data to optimize your ingest, you can use your dashboard to find issues with your ingest much sooner. You can use the baseline dashbaord as is, or download optional dashboards to provide you even greater anomaly detection options.
```

Comment:
Just making sure you'll add the link!

Reply:
Will do, thanks for being thorough on this!